addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -360,29 +353,58 @@ var SelectCreateDialog = ViewDialog.extend({
         if (this.initialIDs) {
             domain = domain.concat([['id', 'in', this.initialIDs]]);
         }
-        var listView = new ListView(fieldsViews.list, _.extend({
+        var viewType = config.device.isMobile ? 'kanban' : 'list';
+        var ViewClass = view_registry.get(viewType);
+        var viewOptions = {};
+        if (viewType === 'list') { // add listview specific options
+            _.extend(viewOptions, {
+                hasSelectors: !this.options.disable_multiple_selection,
+                readonly: true,
+
+            }, this.options.list_view_options);
+        }
+        if (viewType === 'kanban') {
+            _.extend(viewOptions, {
+                noDefaultGroupby: true,
```

Comment:
Probably not useful...

Reply:
@pparidans It is useful in Selection kanban, we do not want to open grouped kanban view in selection mode